.github/workflows/code-standards.yaml

```diff
@@ -23,32 +75,28 @@ jobs:
           echo "bin-dir=${{ github.workspace }}/$(composer config bin-dir)" >> $GITHUB_OUTPUT
 
       - name: cache composer caches
+        if: needs.should-test.outputs.yes == 'true'
         uses: actions/cache@v3
         with:
           path: ${{ steps.composer-paths.outputs.cache-dir }}
           key: composer-cache-${{ hashFiles('composer.lock') }}
 
-      - name: cache composer bin
-        uses: actions/cache@v3
-        id: bin-cache
-        with:
-          path: ${{ steps.composer-paths.outputs.bin-dir }}
-          key: composer-bin-${{ hashFiles('composer.lock') }}
-
       - name: install dependencies
-        if: ${{ steps.bin-cache.outputs.cache-hit }} != 'true'
+        if: needs.should-test.outputs.yes == 'true'
```

Comment:
It turns out that cacheing this bin directory doesn't actually work properly, and coupled with these new conditionals, it failed very consistently. So... yank it. The composer intermediary cache is maintained, and that one works and gives us all the speed up benefits anyway.